src/ui-test/tests/camel.settings.test.ts

```diff
@@ -251,3 +253,82 @@ describe('User preferences', function () {
         });
     });
 });
+
+describe('Additional Camel component set test ', function () {
```

Comment:
what about to make that part of user preferences set of tests? in CI logs it looks it is separated 
![image](https://github.com/camel-tooling/camel-lsp-client-vscode/assets/18696866/c56f2a75-26f4-47b6-ae8f-abd20c3dcb3f)

Reply:
Sure, only issue of immersion of describes. Thx for noticing!